Watch Extension/AddFeedingView.swift

```diff
@@ -75,21 +76,45 @@ struct AddFeedingView: View {
                         // TODO: These numbers are not exact, figure this out better
                         .offset(x: isAdding(feeding: self.feedingIntent) ? -45 : -metrics.size.width * 0.5 - 50)
                         .gesture(TapGesture().onEnded {
-                            self.store.send(.feeding(.start(type: self.feedingIntent, side: .left)))
-                            self.isShowingSheet = false
+                            
+                            self.communicateFeedingStart(type: self.feedingIntent, side: .left)
+                            // TODO: show spinner here
+                            withAnimation(.spring(response: 0.35, dampingFraction: 0.65)) { self.feedingIntent = .none }
                         })
 
                     Image(systemName: "r.circle")
                         .font(.system(size: 55))
                         .offset(x: isAdding(feeding: self.feedingIntent) ? 45 : metrics.size.width * 0.5 + 50)
                         .gesture(TapGesture().onEnded {
-                            self.store.send(.feeding(.start(type: self.feedingIntent, side: .right)))
-                            self.isShowingSheet = false
+                            self.communicateFeedingStart(type: self.feedingIntent, side: .right)
+                            // TODO: show spinner here
+                            withAnimation(.spring(response: 0.35, dampingFraction: 0.65)) { self.feedingIntent = .none }
                         })
                 }
             }
         }
     }
+
+    // TODO: consider making this a global function or in the session coordinator or ?
+    // This is the first time of copy and paste
+    func communicateFeedingStart(type: FeedingType, side: FeedingSide) {
+        let info = WatchCommunication(type: type, side: side, action: .start)
+
+        let jsonEncoder = JSONEncoder()
+        guard let d = try? jsonEncoder.encode(info), WCSession.default.isReachable else {
+            // TODO: show communication error
+            return
+        }
+
+        WCSession.default.sendMessageData(d, replyHandler: { _ in
+            self.store.send(.feeding(.start(type: type, side: side)))
+            self.isShowingSheet = false
+        }) { error in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c7106bd91c271d949fbb3c9cc499512408cb0c64/Watch Extension/AddFeedingView.swift#L112"></span>Trailing closure syntax should not be used when passing more than one closure argument.<br />
<code>multiple_closures_with_trailing_closure</code> <code>AddFeedingView.swift:112</code></td>
    </tr>
  </tbody>
</table>